src/handlers/error-handler.test.ts

```diff
@@ -1,20 +1,16 @@
-const { expect } = require("chai");
-const sinon = require("sinon");
-const {
-  pageNotFoundHandler,
-} = require("../../src/handlers/page-not-found-handler");
-const {
-  serverErrorHandler,
-} = require("../../src/handlers/internal-server-error-handler");
-const {
-  journeyEventErrorHandler,
-} = require("../../src/handlers/journey-event-error-handler");
-const proxyquire = require("proxyquire");
+import { AxiosError, AxiosInstance, AxiosResponse } from "axios";
+import { expect } from "chai";
+import { NextFunction, Request, Response } from "express";
+import proxyquire from "proxyquire";
+import sinon from "sinon";
+import { pageNotFoundHandler } from "../../src/handlers/page-not-found-handler";
+import { serverErrorHandler } from "../../src/handlers/internal-server-error-handler";
+import { journeyEventErrorHandler } from "../../src/handlers/journey-event-error-handler";
```

Comment:
instead of importing these one per line, I'm wondering if we could add a `index.ts` to the `src/handlers` folder to hold something like:
```
export {serverErrorHandler} from './internal-server-error-handler.ts
export {journeyEventErrorHandler} from './journey-event-error-handler.ts
export {pageNotFoundHandler} from './page-not-found-handler.ts
```
so that when we import them, we can just do something like
`import {serverErrorHandler, journeyEventErrorHandler, pageNotFoundHandler} from '../../src/handlers`

Reply:
this is quite small but could be handy for bigger modules in future

Reply:
Yeah, if we end up with loads of these then it's a possibility, but I think I prefer the separate exports for now! They work better for tree-shaking/mocking/etc. and for the most part your IDE should handle the imports for you...

Definitely a matter of taste :)